_posts/2017-11-17-falling-into-machine-learning.md

```diff
@@ -0,0 +1,62 @@
+
+One day, I ended up reading about Golang and Python. I noticed someone was mentioning Machine Learning in both languages, and I got curious and started reading about it. After a while, I realised that those two subjects (Mathematics and Artificial Intelligence) that I didn’t quite like and didn’t mean much to me separately, could be combined to create cool (and useful) things in a completely different area than I’m used to. Completely new challenges! Since then, I got excited about Machine Learning and it’s been the most enjoyable and exciting use of my skills in a long time. The tools I used to like learning (mainly programming languages) became secondary, or what they really are: a means to an end. I stopped googling around, and started an online course about Machine Learning. The fun began.
+
+## What can I do with Machine Learning?
+Machine Learning can sound like a mysterious thing, but we actually see it every day. The amount of applications is countless:
+
+* How do Amazon or Netflix know what you like, to make recommendations?
+* What’s behind those self-driving cars?
+* Is it possible to automate financial trading?
+* What about weather forecasts?
+* How does the Google Search Engine work?
+* Can machines help us improve healthcare?
+* What made Siri or Alexa be able to process natural language?
+
+And these are only a few examples. The use cases of Machine Learning are left up to imagination.
+
+## What do I need to get started?
+It’s easy to feel overwhelmed by the amount of information that we can find about Machine Learning. There is a huge amount of blogs and papers to look at, and this grows every day, as the field is still quite young and the interest about it has increased in the last few years.
+
+As a multidisciplinary field, there are a lot of areas of Computer Science and Mathematics that you need to have at least some knowledge about.
+
+Programming is a must. As Machine Learning practitioner, you will usually be doing more hands-on work than a data scientist, who will be more focused on data analysis. There is a thin line between the two though, and you could even be performing both roles at the same time, specially when working on your own. Proficiency in at least one general-purpose programming language is needed in order to build the machine learning bits, and full applications around them for users.
+
+Machine Learning is specially heavy on maths. Just to start with, you will need to have some idea about:
```

Comment:
specially -> especially: "Machine Learning is especially heavy on maths."

Reply:
👍